src/modules/cmdpal/Microsoft.CmdPal.UI/ShellPage.xaml

```diff
@@ -84,11 +85,14 @@
                     <RowDefinition Height="*" />
                 </Grid.RowDefinitions>
 
-                <Border 
-                    Width="64" 
-                    Height="64" 
-                    Background="{ThemeResource AccentFillColorDefaultBrush}"
-                    CornerRadius="{StaticResource ControlCornerRadius}">
+                <Border x:Name="HeroImageBorder"
+                        Width="64" 
+                        Height="64" >
+                    <!-- LoadIconBehavior will magically fill this border up with an icon -->
+                    <Interactivity:Interaction.Behaviors>
+                        <local:LoadIconBehavior Source="{x:Bind ViewModel.Details.HeroImage, Mode=OneWay}"/>
+                        <local:HideMissingIconBehavior Source="{x:Bind ViewModel.Details.HeroImage, Mode=OneWay}"/>
```

Comment:
Discussed offline, this is a bit overzealous use of behavior here. We can expose a loading prop on the `LoadIconBehavior`, or just use the `HasIcon` on the VM at the moment like we did elsewhere for now.

Maybe later too, we just have a default icon that gets returned in the failure case in case devs are loading from remote resources?